root/static/scripts/edit/URLCleanup.js

```diff
@@ -2460,7 +2460,7 @@ const CLEANUPS: CleanupEntries = {
       return url;
     },
     validate(url) {
-      const m = /^https:\/\/www\.artstation\.com\/([^/]+)$/.exec(url);
+      const m = /^https:\/\/dribbble\.com\/([^/]+)$/.exec(url);
```

Comment:
I guess the list of harcodedPaths below can be expanded too, since it seems dribble.com has a few more static pages now, not that I think it's necessary, but for completeness sake.

For example,

- https://dribbble.com/designers
- https://dribbble.com/submit-brief
- https://dribbble.com/jobs
- https://dribbble.com/stories
- https://dribbble.com/signup
- https://dribbble.com/session

Reply:
Might as well, thanks!